BunqSdk/Json/AnchorObjectConverter.cs

```diff
@@ -55,7 +55,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
         public override bool CanConvert(Type objectType)
         {
-            throw new NotImplementedException();
+            return objectType == typeof(IAnchorObjectInterface);
```

Comment:
You're probably looking for [`Type.IsAssignableFrom`](https://msdn.microsoft.com/en-us/library/system.type.isassignablefrom(v=vs.110).aspx) here? This will return false for types implementing `IAnchorObjectInterface`.